.github/workflows/ci-libshortfin.yml

```
@@ -53,16 +53,19 @@ jobs:
             cmake-options:
               -DCMAKE_C_COMPILER=clang-18 -DCMAKE_CXX_COMPILER=clang++-18 -DCMAKE_LINKER_TYPE=LLD -DSHORTFIN_HAVE_AMDGPU=OFF -DSHORTFIN_BUILD_STATIC=ON -DSHORTFIN_BUILD_DYNAMIC=ON
             additional-packages: clang lld
-          - name: Ubuntu (GCC)
+          - name: Ubuntu (GCC 13)
             runs-on: ubuntu-24.04
+            # Only test with GCC 13 and Python 3.12
+            python-version: "3.12"
```

Can you set the gcc version explicitly here too for consistency? If we change the ubuntu-24.04 image and the default compiler changes, this job might no longer test gcc 13.
```suggestion
          - name: Ubuntu (GCC 13)
            runs-on: ubuntu-24.04
            # Only test with GCC 13 and Python 3.12
            python-version: "3.12"
            cmake-options:
              -DCMAKE_C_COMPILER=gcc-13 -DCMAKE_CXX_COMPILER=g++-13
```